hack/generate-manifest.sh

```diff
@@ -311,6 +317,13 @@ if $ENDPOINTSLICE; then
     HELM_VALUES+=("featureGates.EndpointSlice=true")
 fi
 
+if $FLOW_EXPORTER; then
+    HELM_VALUES+=("featureGates.FlowExporter=true")
+    if [ "$MODE" == "dev" ]; then
+       HELM_VALUES+=("flowCollector.flowPollInterval=1s" "flowCollector.activeFlowExportTimeout=2s" "flowCollector.idleFlowExportTimeout=1s")
```

Comment:
I just realized that these values (for `flowCollector.activeFlowExportTimeout` & `flowCollector.idleFlowExportTimeout`) don't really make sense outside of tests, yet we are applying them every time we generate the manifest in 'dev' mode.

Maybe you could modify generate-manifest.sh so it can take as parameters extra Helm values, or the path to an Helm values file. Alternatively you could generate the manifest you need using Helm directly, with your own values file, and without using generate-manifest.sh at all.

Reply:
added a new param `--extra-helm-values-file` for  `generate-manifest.sh` and defined these test-facing values in a separate file.